### PR TITLE
Adapt `NetworkPolicy` rules according to recent refactorings

### PR DESCRIPTION
**Pull request by timuthy**
**How to categorize this PR?**

/area dev-productivity ops-productivity networking security
/kind enhancement

**What this PR does / why we need it**:
This PR adapts the `NetworkPolicy` rules according to the recent refactorings in `gardener/gardener`
See also https://github.com/gardener/gardener/pull/7589 and https://github.com/gardener/gardener/blob/master/docs/concepts/resource-manager.md#networkpolicy-controller.

**Which issue(s) this PR fixes**:
Part of https://github.com/gardener/gardener/issues/7352

**Special notes for your reviewer**:
/cc @oliver-goetz @rfranzke @ScheererJ

Depends on `gardener/gardener@v1.66` to create the expected network policies in the extension namespace.

**Release note**:

```other operator
Adapted extension components to support the [FullNetworkPoliciesInRuntimeCluster](https://github.com/gardener/gardener/blob/master/docs/deployment/feature_gates.md#list-of-feature-gates) feature gate introduced by `gardener/gardener` v1.66, see [here](https://github.com/gardener/gardener/blob/master/docs/concepts/resource-manager.md#networkpolicy-controller) and [#7352](https://github.com/gardener/gardener/pull/7589) for more information.
```

### CODE REVIEW

File: example/controller-registration.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ metadata:
...
.../x8GcvY/dq/8heSczd/Eoc5ssPDAOkx2RfD4tt2UIuMGnbkgx1XgdGXmwh+JqTTmDCClovRQplpBW5QZq98czaYJkZNcd+WJrn5rKHklLgHtPUtdqmau3xWGv34VZ0ZnQbgs+d3mVaLMFIspihb/YHOlkqs/lCtWNcllvCJfD0xf8fdILxVVlZ5if03nqkgR94o0Axhbomc203x/2N/J+HeFavJfhl/dVvBL2CD/D48P8v5fB+36978eBsT9fC4V1W9Dw/o9Nid2iBI+GR6wQMUt2CShpZ8mFkmDyJp00ri/c+1q/+D2MoiuRAzbRiN7vbmjyWe84dxRZySNR7AmjjBYrgjQKBfRMr4SUxtV4yXhWzqpA0pD5kTM2V2t5P7ooyRklMybxs11fc1LS7l3NNSi/uv9r/cbjfTgAr1KGyDd0rocrZJVvfNoRYc8ymiQg/bXF+5GydpEZdtsNLSL7Jgp5wrQIceQWHrBXRmraDsn7gSJblxFlBwydcQF/qLb/R1yuE+Arvxvs3d0nhd4jUPDuR+LzCECMnRkNeiTbEeKsYKtPCnpROUcHai4m4YM7xncEj5URexwx2cydHiBrhO6UovaIGJhkl+DqQ+l6H/TtxaYxDw2l8yhHH5UQGQsbSz22yYMNtOjEQx3O1zOIzMIJy0HdDvu74h4W4/0EDoYSD2Iga8n+wzejEbnfMCfXQ73D/TQPklJFQ+mQ/4F31/JwLY84iAWPKfRHiN9UbmKValKg4GD8G3/5eCSiM/rYZdcXQ9uuqM++aH/E3+f5DbNXMn+5VlxCezjoiBvRAY+e46fR+YRz2hbvXSQMNEQFaFXHsJjjAltJDFyP6U+ifFHEJLoYCpg8sJ1eBc+yjSv178eDb4b9IBM1ahHqkWPtOYU5ftwyBpEDEIxqtWPISlhwSchIAAzFof3c+EMxqOoigAbS3XoiqKJx7BAZOmdEj5tLJClyJCufhzHkhmTIBfVMhHM2TD4iE1LG5nTPF5hyU/4NBqrjjkd8vMvKLWL7r7jjzXhCBKeFx3+nLmen/7+T0NIRCE5rzUdNXGjaTzG+ZfqpOKyFTVbWdsEEq7Omvtm2/wriFpVuNNIfl+jgxcjPrU2r6GGGmqooYYaaqihhhpqqKGGGmqooYYaaqihhhpqqKGGGmqooYYaaqihhhpqqKGGGv4M8P8VLKHWAMgAAA==
```
Comment (MartinWeindel) [inline example/controller-registration.yaml:8]: Fix controller-registration.yaml as the chart generated on Linux differs from that on MacOS.
```suggestion
  chart: H4sIAAAAAAAAA+w9a3fbNrL9Wv0KXLU5TtKQsuXXru5N96qymvrUdnwtxz09TetCJCyx5usSpBw12/3tOwOAJEiREqUkTrM1msoUCAwGM4OZwWOgCY1s5rPIYG9i5nMn8A0+DYLYsFgUG5xFM8dinc/eKW1DOtzfF38hlf+K553dvZ3ufvfgAPN3DrcPDz8j++/WbLOU8JhGhHwWQaeXlVv1/hNNk0b8N6fM9ZyJH0RsgzaQwQd7e7X8B7YX+d/d6e7ufUa233tvK9JfnP9fkHMaxyzyOYkDIllM7qbMJ+PEcW3Hn5CQWrd0wrjZ+oJcTh1OeBKGQRTDA4iFSyZuMCYeja0plH5GIubS2JkxqBdPtXzq2wDAZxN4G/jkcRixG+cNs8mdA+X+64lJXvrunAS+qIkokZBFxHV8ZrbMo9H1KAbcAMQg8DwAcDUYEduJeMucOHFHfEr0W+b496gjPtOM6aSDH+lXPvM7OaAx9C8JyY3jMt56avK7ED7H9BY+Yw+e/wVFr2jkBAknx0dDaDCMgt+YFbdMx2a0I8tBVsuccSuwWaf1sbnaPDUb/4MpjWJzTj13kzZWjf/uwU55/G93dx/G/30kGjpXLEK+98hsp0XDUPtq7m6b24bNZi2bcStywli86JPvwBwQC4WC3AQRiaeMvFCCREYoPmQA4kNGUnxIJltmy6ce65FGUteaVWHysQn2H5aajX87sMxJsGkbK8Y/jPaS/9fd3jl4sP/3kjodMIPhHCzlNCaPrScEaP93Muqfk9GQwNCmvvhCb8A8OjRmxAq8kPpzk/TB9ItqHEw+CgqzTekfoCUl8NcF0fE5WPjEt5nUEn1wJuDPKLiJ7yh4GieyyDMyM0kX9ITFwphQTvwghnoBVInuHA7QfFH95HgwPAPEsIVWpwP/UggVjWSwlUYjXXObPMYCbfWq/eS/EcQ8SMBPmWOjJIHG4qwTCiFoHbsNBPBBmwl/Jc4bMBHGjwpGMI4pFKdQIYRvN3pBQmOFtEjTOA57nc7d3Z1JBcZmEE06imi8o/pqANaq1isfPBSk9v8nTgQ9Hs8J6GuoQMeAq0vvBMMmEYN36Mz55C4CpwidL64IjmBsh8eRM07iAtFSHKHregEgG4hAuz8ix6M2+aY/Oh49QyA/HF9+9/LVJfmhf3HRP7s8Ho7IywsyeHl2dHx5/PIMvn1L+mc/ku+Pz46eEeYgJ4Gc4PRBDwBNB8kJEoOwRowVUEhNCg+Z5dw4FnTNnyTggpJJADbBF04pizyHI1u58CwBjOt4TiycS77YL7MFRSZBb4LKDuWYT4lhkbZpduDfjPl2EHXAhZwmYxN43Un1Yv4wBTexk1Y3rMCPo8B1QXNGbIIEEy2bAHaZJiUm+fKxRWMi270aXoyAWk/UV/aGAllYpw44OmBkmILvLYJvYy/BoxcOuzLQzEfx4KTQeenBC0qrTCQq0ssKogh8W5LjQAo4tEId+oM5/sRTM/sfM5BLEBy+0UrQ2us/4BEc7j+s/9xHWpf/1zDlB93LzThsPBdc4f/tdRf4D18e5n/3kt6+NYjNbhwfvCKcm7WJ8ccfrWbzM6wLhlPUaOmAsBTYjxtnIsEZhtHSJ5qLsMysIW7KmmaKg2m5QWJ3ZjvUDad0p3Xr+HYPnFYslEij1AJPIGHRmZhbvn2be0ht84q6CQOY0Ba6EiDDsqYJuNLEjY9FTRO7jq5PWrNN1qoJfQTHBpwmS2VviEYKBOpshIxWH1BCljg3qytfQDtQkZ+z6IjO/y8JYorV69+K3m0KNpWaJggKQZF941hB/95rEeXb2Cvw0WuZqkoJE2qB6CBAjzruhsxDGKYAsBnztPqAESFiTrWpJAlgEsI7YKMACHSaSpNeP4yCkMoF30vHY0ESS1iL+c1EqglsjaubIu3MoMD3bJ4im37vkX+21kczg/ZPAoPTgz+gwZgfk71NEIYZFEwVrVtUdpI9PEVz4cUKotaCWg8jiw7yPAWhmNcTQOLgR5y/NAaU0am7GWOpFQvS95N4GkTO72pqKKrXve0BixJWbEx7FM+ZmXM8mAdJC5chNKX8XGxskDaf0u7+QS8fcqK8GdNJ3gWQDj++Ie1H/H8f8XLJiIUBd+Igmi8DwVzOqgD2NgZY3e201y6jMLdmLkwSgWKOLQmwzFEwZBUjraM7Dh/b//mrp3X9f5GtPLSm+0Er938OD0v+/97OweGD/38fqbj/87nyrkfMilj8ectjMbUpOHyfg26QWzfL5ES57qosD6kl/XHzAoY+5cw8S7NBq3zeyiBr8iTKO77lJnZpImGCQRgf7DHfSg0pqJ2PTb5PPq09/l0YMCy6cVz45IYbTCaOv0IVrBj/O7vd8v7vwd7+w/7PvSR9/N+AR+DHYyc25ZPpBOm0u5tOuyX7vxXsz9UDjGGXjsETwSeiAcpWz+N5CLqgzRnMQTJdkhqTdLHboz54I5EugTLLA3AG1m3hngA2ogQQH8HpoRFIqmybkFs2lysBIJwqSxUQDaaNyDxVQO1gHWFnUhdQHF3pkdtkzCKfgfCbT2sQflqP8NOFxY8/DZUjhttDzADnr5qsqsAlnaSkjRKX8fSLQba+RMfxl5xCr80ZjV6bQPjXaAtwKwygvTYf1+FgfvWPa/y/noBPcEsO1RO4rsw2v7zsvxAM2qrm0INFWC9tqP/l4Hk/+n9777C8/nuw9+D/3U/aTDOdS92pa6Zc1ywo2HWUVqqIIjZhb2SV2PHk4gs+ZDnfBhEMf6j36EfjkWc8si8ffdd7dNp7NHr0e7sldTqCIFu/YPHn7cf/+B98+Pq1/XbvDwM+u+rz8qdf2j8/fdJ+zb9y2Yy5z6EkhwfQffOvX9999eQ19/hEAABx/9qEolv/MWpm7fEvvHGPhoZYOZjBjD6IDNyWF7aiUhGsGv/7B/vF8b8rjgQ+jP97SKV1NMHUK8HUlylPxfp48ZigvgdzSsMKRaDP4uS+kglwqoWm2XyxpEhoGJqa4QdVtazhmiqOD9z3q1oVGyCqRwJpfl0U8tJacCXhiiu9+YLax2a6ljYf/41Mv0ir5n+HO2X7v7+792D/7yW9r4GtpMGoWgGSMlYav9qGb2mn19K3dsUsDiFno1FOUQxtIGMFHI4I9qdvj08uhxc/qzkKIWIwl1Jh6kfIKU4jSkXeddaXwgHH5bqMQj4vJUR6UsW2NSfqvIjphZymXuM8Nc27xNlq876rSd01zvqaEiBvH+Z/ZYAfcQ6I3VYzkVQIDEmJ8/7FqI4S9eSVPmVKJ/AeNcLDN/X8AfzJrB3cybzGvUJCcnc3e6Hwq/B4N/ZH19b/kc3XDQRZrv+73d3D3ZL+7x4+xH/cTyqvTsGzpplv/8blJDA1DECtwAMtFCSRxY5wR9ARO3qLdsJR5y1QjkoavqEn1+wQUjWE5Y5ddR2pE2xjPC/WSqNY8PyJmp1OoiAJ5Wx3sW/C8smuSarJ4wAiw3V4/L2WeQLfxYvQBYPnZmQTedDbKD7LoeGSm5PD4mAVE5dGvTyTWwHOpTO3GdFRUTQKhkGobQueUfcc94pZNAjcxPOzNrBMMdxncDoUoTyapiTkNx745zSe9oiJVNHPi2ilJCdBFV8NL7RsOeXHs0r+ZGmzF9rRY3k4ZxUK5UISg+Fp//hkTQRGMY0Tjmf49QPQNe2LsuIPW+z+Zf/y1WjN1qV8iKJLmywVkC1e/ng+XLO9YPybOPYdMUlstD7QhBdWt54OeDOtcFlRXiLTf7GIi63TqYSJci9tYgfAS5+v4rgqZqp6C+0fnw1OXh0Nj66PXoIUnOmMCCMnQLPcI12wRMspJoFlxxBlJremIHD50IHh5/fPj692R6UXpJK7Dhdn7uX4JYMLs4AbAItih3EdCiquTFsX8kstbCEaKvBFHt2QbSl1AOSVuMsYFXFEDIMzwOUqi7no1w2GgUgZMZUywPjfIHFt3L6dYbhfxKxg4ju/Z7BFTDE2KnyHuARTKB9QQ2SGU2gRICxCaCImJDHxNXgzOcsmpxie7Pg3QU+Ez/BeB4M2UjtlBZ6XgEWaywgKDGEJIt6x0QvrcGdi0MiagvNrxUnEOkBIQ6Dui6M/pmd/ESnLxrdKuFaOIExCwy/lA6p7ZDRV1WVfcnKngRcXw9ElSREQLCnzQI7QrCLPGYFkA6qIkBeMXYoCT8Bkvh0GQGcZ2eE6UKsElCdjz4nlAUE8rAkcM8mA+hgLNWYkCXGk2iY59iHXY+4ALeKHZgNSmxtI2uaM0D2QxQqSdoVXqS2v5ZwcoyMolo5TrJLGdMkxa5YA1A1aTOkZ03JatHqqOfEMEOPACtw8Gkpovgo4y9oW7QMbdHtaXaqMT6lSituNC7M+50YAhTkJoFVjJItJaQyQLBG5huecb4gPUzwMBauuIhk4DgLwxqoBKwPQqD9HsiwoaukDufOUsKAfFSAC2cEdBr9FGCE1dmy5fFGdMIZKO36YjaSaCqu4hIm9EZasvkCpU0NZnmBEpeyDdEIFbndTByb3GoZLoJJcDyCsrPPksTSuqKuoP0etoWj1ZAk0GOLekl5iqh3OVcVoFNF5bSll/hsT7dj/MERLhedPT7JazZgmeRi+yagapqfehdERX2CaIWI9wQNIVEwnLuKUggk37CLOCNF16FtWkPjxN2ApoGgzhQZKdVhZXRrpiIEZZRjnC5hTMugvGaXKf6ESTGoZhNqWM6G6DjZRArdsfnzUWJpF6ZQD8KRwGfTxDTxS6QOkPV8qYYosCG0MDxhMXNcRTI0EErCQhwov2M06ncoqZTZYZCCbUudSFlnaIzmcp4FrKyBzD/yFCKzWaX8gKKToldInpcFSqKnciIBw9D0EpK2pR63v2XwLo+EzfPENxwtswOLV2xJBzqxL5OlTL+Hx06fCE/ONEMYUs5+RMfhgB3vk1cUJOHh44YwtWl/GoyYCh0lMcpaWKLHoTAVOgb8HOlCQQoS/Z8v/yKVsTK2ATNBJTvkrHeFlncLUUBvq+xHrdk92Q59DyRzVVyFbKzumia6IGEPGSrYi3d5TL1dqdJIFAtVTwZC6ZPn7bFhualuqolWaqO/zipiVbGNWckcF4WRN1PbE12AkeIWEcOWwCo7qo7ORHqdkprsDv04DHv9aCxRAiBI9DLD/9Rlh5sQk7b+Z4r824ch8mBinOb393TbWaU+CYOIyI0zGrmMZts8Naso8vI8Ai9UJyQqfoYHsrPIV8himlTq8xKyFevVqHM1mbScUBih672JTV6m3D6Xa1lJrjQZ7I3W2kSrD7KWd2UyNNRbDJUpDRUQ2Eb1RFvqIvURT2dw7W4kqv3VCUA+DKfj6zJ+wK+o6dvM59ai2OvFodMtTdw2Q15b7K8EGTNyVg2tKCIIJrWWlgLlYtxGALPAZoKR4z3BRhvEOrq9Fdt1ElYjZjFjeVaqRitoZ2mLBrqQlkeCIj9Cim8/pl1spY2EnQH+3sAdRbLRGvKyFhSNMBbaBL63ECZ7eeTXmXnQqIioyH5Rm2pO/mNJc+jqqCO1fNQqqovqV/Hn0jeMlHvETbwy6FxTuqgWUbOEEb/q06TxfepN3QC3TgLJnuIkwWXhfv+grNs2aLPvKzb90ZGVbgfe59JvrGLnVty57CXmjHZMyQhXkYiT+rR/c+caNw2BCnMW7FNPmCvEjIAuDlNPJahKfynJltkaERRF8KjD1uNeMxGAs7797oa7SqvYDCpi8XKiSIpUCyy7mgldS8Kp9kRr7c6POx8MAOdhbc/i8d81ArTih7p9IQUhNsFpgUA0IcUlvsPPCeP6MbJ0zsfSzBY9DFB14AHy2Lhi151try494uQqVSyhU2jDGeiYZJBEY01jcnQwfW6hstoRztmXR8v6ZTOjZqTvgqp20JejW+2YGKZ9/yEEtjPpqMIaQ59bS2nJ4FDQBrqzh+C/kJeNsOzdvRBkA8vYhdlikdc//2Sx0gzmezGx+CnDF+e/D7b3y/e97O/vbD+f/7iMtnv8LeX7e7yjjdsUJv6bn895fbEfzE4H4ewUmn3bEJZUf+ijh1JlMDTqDKSkdO64Tz9MLMjL9Uz7iLrVafhZaC3ubOYjedw6qtPkJXmnaI13xRtz0WrzTR2UOxHaNujTKFUEokpAiQPZEo+z7oO0mREoViEKrdFZDHDdRF/BkmlosIfDE68ipU/FGGUng5ZdFyPt3AELKJ0wN450WkMjjXnL6VAYzFXB6LK7jIV+al6r/5jdAFzzBRtqNQunaT5Z3JbsPqbJ3co8rvfNsZEUw0v2JXgEmLVBoyhJNEtoVLGzX1OAIE+ug4a0rhG6GAltCTFxCq0PP+wRfVK5bkN/3I8GbDXRMUPwuiHCHYGFQB0YE49DxmAF0Tm9vU/5qs/o23kCl19APKaWHFQcu5fys2GU+5zDEjL9nhxhVd9U25tnaBMojN/Kzx5uQWYh2cZyqa7NMnaoi7zxx3fMAFNq8oOTkFVVh9rIwPgPPo/oJPIN01kPQIIbSJs87LLY6Bb9Hu5imUAEmMljJkp63gQupvoW/H/Jcv3AtU+/iWRUezX2L611AeFNG3XgqlM36sLXKDdqJ4jGjsZEZ5ed1NpnU1ATg7M4Qhzdn1AVSQZv2MuRkPVPUO1bVRrLWQjO2w1FVaWFCBcjq9SB/C8rxt8DxSftZu9xZ+WsvBq7KCNti5GamDltZ5WVao59VKMMu3WhmOPZzXcQX70gzV0FIfQXDDaxbHT9ZcKjKZT7FCRQrw8wkXx3y1cFkyka90+syf6aPIDnOT4b9o+HF9fBkOMAb3a/P+qfD0Xl/oG/SiaOs34KyL87ixFpNxYK2yJfnt7Pz45kgbmqiU3yPT/svhleA7MuL65dXw4sfLo4vF3DtEekSatG8ncrw3jWsbObl6WW0qScwoXT1YV6jfCXkzvYKCz/DYA12ikqdL/KswkfSKOBhLUn+BVVXmJlTG3+KqbTWtiFbVrhLNegtMKkZfgViSVIt2K9lNJKupi62MudsjavP1qbVupSy0uhcHc/a4NwlQfdpUgcmTgMbQOx1t7VOKIJ+7FnqQ/pQad31nzCwwQhHiViUHCf2hK1eCFoV/7992C3f/7O9/xD/fy9JaSrOvBlGAnohLg23v35OdszujrENCmRAQ7ms4YAm+x5mTGqtyHzhxFe5L6EvIoXCT8dlJP2C2soSY/ADdvS7ZuXC03lgH2Vi9o0Qs09sBWrd+WU60wP3/5WvFpNcKLvzZ13U+diS+5DeR1pX/0djar3f+P/tvb3dsv7vHhw+xP/fSyqv/wv+Uv1O9MVLAOQdcBeBy95BJ3+i6/3qBkwDw3Bf4C0A6q7M2utsWoW5IBZVVyjiuTuwuWOVOxEhDIa4IUA83KGaf6eG8qKlrx25C9oIAXm/avok40EXsarb6VhESt2ykOUWkWg/bS8Cb7cXwWQmfUMyVt1zUcEqcd9Fmmln913UNCoPS+qEyqm3SLK6kVYrMVHgMl7OGMsgFJmflyi9WhNZGZPvsipWV3GjLafJvC2/MQzmFV9Wshb3G6sA5vvMZTjvir8VBBGQZTnFxaAvEU5rEKWL5OJF3rWVND+77MMgjZYTjSpNly25FtHXiIQkXGRFxW0mFZzJLyjJntdRJwvNypnAYjsVE816fqyju3TZeCfjp8K//oI2EDqvVndTZi6hXSu9Bkf3G9ajFE/E+R9hd9MfKNC3tz7IhPBje2b3k9b1/9PlznWmACv8/+72Xtn/hy8P6z/3khqfGKi8KFINww++NLNwQETbRa89ZIO3oBjSvMAf2xDa1sjWUnCfXdwuaMhDDAbIgTIy68NXe/YGHmjgPbL109s2PrZ79YcenrXTqz3avfbl4Lz9x89bjdvOyJb1R7WqLQ5B4+3SDWf6phu0KoG2/1in5UVK5nvJQAYHTygVJh6fqJFL1+PkUS1lu47PIcdKn3vkLPARuuR6Ni8Ru5vIatGwPP1SLweykBKFHgFJaJVX/O5/NfJh7+cvkja0/+rKi2ZuwMr73/fK93/u4+sH+38PqTwFKpv21MP+tDZf7s1KiOuvkEBFcl0Gtwx/UIO6nP25Fei6438W0nWX/1fu/3Z398vr//sP5//vJ5X8f2Sv+oXkks/fRlHnFnh44B1mqyK4fdttK5VxhcFcUOI8sPuqcOWPxDSSOQNQ+Xd719LbNBCE7/srVvWhEmpcJ21R2BMhNSiihCqNkBDikCZuYkTiKnZKI/Hj2dnZl+ONGiQECOY7RO16H157PbOPmW8OlB5mqhboi5nG+paj9TQUOdbcVSXmvtWQvYRGQMfPnEvVMl/1cOLvm+Iss2Wx3u6J29y8qdjVEmPRcMDmg0o2A+XiqsYa44V8PSC94e/hjIoOlZ64v+Y/VUxBuyJvAgw98TPHLt9fGyfjX8Vh8l/Tr/6s4Nd4Qv6fXbR3/b/aHYr/9XuA9vlKKprY0HL9vonn0zVIeDs85AIVtmBtwql/mhiSBtVkLhzv771n2j+4GxbVNXLYMlY3bxaefAYLZ2HOSFgk18QVkOUiQaNeRGt+pdJsVN1uudrScQ4oTOeEmuu7WtZ+NLKUUTqv483NV56XlnHvYGZR3026CWPu4AK8SpmUbq6tmddIU+9EDR0S1TRIu9N9lz8pWY9A2R4x5hmyQ6YdVwDBL2TiXgN3M1mFubN1J7C6sVmRPWQSaMAfsu4X/Czh8r52Y7ML/5kHvMZlx5Ufi86BhAxCNwM+ydPKPFh8rCrJvkTjHF0Y3s2Wpvcs7rgaqsgdPluVmjo8oOtQV3qsDciFyb8Ui5Uslb3MHidySqy4uXQO4/BjCJGhdOsh6cRysMVfs0oO9+l6e1/FxXp+OpO6HfZ3kG8r8il0gEi92Mjn+jwp5ZXx+EoN+MvhTdL2qX1sScMHI/h3+f97TWyrGAeh4FVWHZc8xcYNV6UpLSc4gFfpm8GQ4+/opsevR4MPvXHK36Yf1XWbO453SqbDy3AJeMchkjeuic9O4Pc8PlcZp5O+GyQldsQw9OpDeOCY8EZSyb8tshXfQBAEyw5mCJMf8pl6hVGte/10NB68HvTlbZpORaZHkdedUL5fVxnjOAhxVJtgSEZYqI9QViCnsTC8T9AZTLGoIsHG1hy6gmhSHBZQmbMpUZ/NRMpSeCA9/ziutF+MrRybLZHMudVSI9aVbtVO81SDe0L4MNZ0zBH802eQ2iHbdwjWBCMIPS+E+rtmnu/i/zCUiCg5R56OmufVYnML35/TSeGyB2q2fX3DSpQ6O0riTvxCilpTWDAbX0OAYcSf1uYEAoFAIBAIBAKBQCAQCAQCgUAgEAgEAoFAIBAIBAKBQCAQ/gf8ABUsodYAyAAA
```

**Review by MartinWeindel — COMMENTED** [review]
fix controller-registration.yaml

**Review by MartinWeindel — APPROVED** [review]
/lgtm

**Review by rfranzke — APPROVED** [review]
/lgtm

**Pull request merged**